recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,4 @@
 libopenblas:
   - 0.3.20
 openblas:
-  - 0.3.20
\ No newline at end of file
+  - 0.3.20
```

Comment:
You can remove the local conda_build_config.yaml because openblas 0.3.20 is already in the general cbc.yaml https://github.com/AnacondaRecipes/aggregate/blob/cdeb46a147a8c7ab48f30406200145b00d31c5ac/conda_build_config.yaml#L165

Reply:
yeah, recipe's cbc (conda_build_config) seems to be no longer necessary

Reply:
otherwise, recipe looks good to me

Reply:
removed and repushed!